.github/workflows/gcp.yaml

```
@@ -54,3 +55,21 @@ jobs:
           CLUSTER: dev-gcp
           RESOURCE: .nais/dev.yaml
           VAR: image=${{ needs.build-and-push.outputs.image }}
+
+  deploy-prod-gcp:
+    environment: prod
```

Ser ut som det er mismatch mellom miljønavnet som har fått beskyttelse og det som brukes her

```suggestion
    environment: prod-gcp
```

---

oops sant :+1: